subsys/bluetooth/host/classic/avrcp.c

```diff
@@ -0,0 +1,369 @@
+/** @file
+ *  @brief Audio Video Remote Control Profile
+ */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ * Copyright (C) 2024 Xiaomi Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <errno.h>
+#include <zephyr/sys/atomic.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/printk.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/classic/avctp.h>
+#include <zephyr/bluetooth/classic/avrcp.h>
+#include <zephyr/bluetooth/classic/sdp.h>
+#include <zephyr/bluetooth/l2cap.h>
+
+#include "host/hci_core.h"
+#include "host/conn_internal.h"
+#include "host/l2cap_internal.h"
+#include "avctp_internal.h"
+#include "avrcp_internal.h"
+
+#define LOG_LEVEL CONFIG_BT_AVRCP_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_avrcp);
+
+struct bt_avrcp {
+	struct bt_avctp session;
+};
+
+static struct bt_avrcp_cb *avrcp_cb;
+
+#define AVRCP_AVCTP(_avctp) CONTAINER_OF(_avctp, struct bt_avrcp, session)
+
+K_MUTEX_DEFINE(avrcp_mutex);
```

Comment:
Same here. For `avrcp_mutex`, if it is internal only, I recommend defining it as a static variable.

Reply:
Add static here.